src/utils/filter-properties.js

```diff
@@ -0,0 +1,12 @@
+const filterProperties = (object, ...properties) => {
+	const keys = Object.keys(object).filter(
+		key => properties.indexOf(key) === -1
+	);
+
+	return keys.reduce((acc, x) => {
```

Comment:
mmm Ok. You can do in one step (filter + forEach/map) but reduce let's you create the object directly...